### PR TITLE
Web rtc connections

### PR DESCRIPTION
**Pull request by RetroPronghorn**
**What this PR does** 📖
This PR adds functional WebRTC implementations to the app and utilizes them to make use of the typingIndicator

### CODE REVIEW

File: components/tailored/core/chatbar/Chatbar.html [hunk 1]
```diff
@@ -22,5 +22,5 @@
 
     <TailoredCoreChatbarControls :sendMessage="sendMessage" id="chatbar-controls"/>
   </div>
-  <TailoredCoreChatbarFooter :text="text" :charlimit="charlimit" />
+  <TailoredCoreChatbarFooter :text="text" :charlimit="charlimit" :typing="recipentTyping" :usersTyping="activeFriend ? [ activeFriend ] : []" />
```
Comment (stavares843) [inline components/tailored/core/chatbar/Chatbar.html:25]: typo - recipient

File: store/webrtc/state.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { WebRTCState } from './types'
+
+const InitalWebRTCState = (): WebRTCState => ({
```
Comment (stavares843) [inline store/webrtc/state.ts:3]: typo - Initial

Comment (RetroPronghorn) [reply store/webrtc/state.ts:3]: Let's open a new ticket for this, doing a search in the app this is a global typo

Comment (RetroPronghorn) [reply store/webrtc/state.ts:3]: image

Comment (stavares843) [reply store/webrtc/state.ts:3]: ohh got it, thanks for checking! 🔨

**Comment by stavares843** [comment]
typing indicator is pretty neat! 🎨

**Review by stavares843 — COMMENTED** [review]
I noticed when we select a emoji from the emoji picker, it doesn't show the typing indicator 


https://user-images.githubusercontent.com/29093946/144728978-1c8456c4-4fc2-477c-93ea-1e3cbf33e111.mp4

flow
- open 2 chats
- on one of them, write something
- typing indicator appears
- on one of them, go to the emoji picker and select a emoji
- typing indicator doesn't appear

**Review by stavares843 — APPROVED** [review]

**Comment by RetroPronghorn** [comment]
I think that's fine since it's not really typing per se

**Review by WanderingHogan — APPROVED** [review]

**Pull request closed without merging**